example.mjs

```diff
@@ -1,28 +1,29 @@
-import { start } from './index.js'
+import fastify from 'fastify'
+import { restartable } from './index.js'
 
-async function myApp (app, opts) {
-  console.log('plugin loaded')
+async function createApp (fastify, opts) {
+  const app = fastify(opts)
 
-  app.get('/restart', async (req, reply) => {
+  app.get('/restart', async () => {
     await app.restart()
     return { status: 'ok' }
   })
+
+  return app
 }
 
-const { stop, port, restart, address } = await start({
-  port: 3000,
-  app: myApp
-})
+const app = await restartable(createApp, fastify, { logger: true })
```

Comment:
```suggestion
const app = await restartable(createApp, { logger: true })
```

I think this API would be better

Reply:
@fastify/restartable should not depend on a specific version of fastify. That means users should be able to pass the fastify factory.